src/containers/Menu.jsx

```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react';
+import {Link} from "react-router-dom";
+
+
+class Menu extends Component {
+
+    render() {
+        const links = this.props.items.map(()=> {
```

Comment:
```suggestion
        const links = this.props.items.map((item)=> {
```
`map()` принимает в качестве аргумента функцию, которая в свою очередь по отдельности способна обрабатывать элементы массива